exonum-java-binding/core/rust/src/proxy/runtime.rs

```diff
@@ -116,7 +113,8 @@ impl JavaRuntimeProxy {
 
                 ExceptionHandlers::DEFAULT(env, exception)
             }
-            _ => (Error::OtherJniError, err).into(),
+            // TODO(ECR-4015)
+            _ => ExecutionError::new(ErrorKind::Runtime { code: 3 }, err.to_string()),
```

Comment:
I feel that I'm missing something here... @slowli am I?

Reply:
I've answered in the ECR-4015 ticket.

Reply:
Thanks, I will correct this code